anesthetic/plot.py

```diff
@@ -246,6 +251,84 @@ def plot_1d(ax, data, *args, **kwargs):
     return ans
 
 
+def hist_1d(ax, data, *args, **kwargs):
+    """Plot a 1d histogram
+
+    This functions is a wrapper around matplotlib.axes.Axes.hist, or
+    astropy.visualization.hist if it is available. astropy's hist function
+    allows for a more sophisticated calculation of the bins. All remaining
+    keyword arguments are passed onwards.
+
+    Parameters
+    ----------
+    ax: matplotlib.axes.Axes
+        axis object to plot on
+
+    data: numpy.array
+        Uniformly weighted samples to generate kernel density estimator.
+
+    xmin, xmax: float
+        lower/upper prior bound
+        optional, default data.min() and data.max()
+        cannot be None (reverts to default in that case)
+
+    bins: int or list or str (optional)
+        str only works with astropy's hist function. Then it must be one of:
+        - 'blocks' : use bayesian blocks for dynamic bin widths
+        - 'knuth' : use Knuth's rule to determine bins
+        - 'scott' : use Scott's rule to determine bins
+        - 'freedman' : use the Freedman-Diaconis rule to determine bins
```

Comment:
You probably don't need these docstrings, only a note that kwargs and args are forwarded to the appropriate histogram functionality. Whilst documentation is generally good, if it's liable to decay/outdate it's better to not have it (good documentation >> no documentation >> wrong documentation)

Reply:
deleted accordingly